### PR TITLE
Sprint12 04 a45616 db postgres cb review

### PR DESCRIPTION
**Pull request by Neve**

### CODE REVIEW

File: cookbooks/db_postgres/attributes/postgres_tuning.rb [hunk 1]
```diff
@@ -43,12 +41,9 @@ def value_with_units(value, units, usage_factor)
 eighty_percent_str=value_with_units(eighty_percent_mem,"M",usage)
 
 
-if !attribute?("ec2")
-  #set_unless[:db_postgres][:tunable][:shared_buffers] = "64M"
-else
+if attribute?("ec2")
```
Comment (ekargard) [inline cookbooks/db_postgres/attributes/postgres_tuning.rb:44]: This is a logic change - looks safe enough.  Have you tested this?

**Comment by douglaswth** [comment]
It looks like the YARD `@param` syntax is reversed here too. Please fix.

File: cookbooks/db_postgres/attributes/postgres_tuning.rb (hunk 1 above)
Comment (Neve) [inline cookbooks/db_postgres/attributes/postgres_tuning.rb:44]: no, but it is very simple change

File: cookbooks/db_postgres/definitions/db_postgres_set_privileges.rb [hunk 2]
```diff
@@ -48,18 +48,18 @@
       # from group role, see: http://www.postgresql.org/docs/9.1/static/role-membership.html
       # cmd ==> createuser -h /var/run/postgresql -U postgres #{user_role} -SdRil
```
Comment (douglaswth) [inline cookbooks/db_postgres/definitions/db_postgres_set_privileges.rb]: This comment is at a funny indentation level, perhaps it should go before the case statement?

Comment (Neve) [inline cookbooks/db_postgres/definitions/db_postgres_set_privileges.rb]: hmm this is comment to "when"  statement i thnk.
Will move it before "when" statement

**Pull request merged**